test/components/helpers/component-testing.coffee

```diff
@@ -9,9 +9,11 @@ ReactTestUtils = React.addons.TestUtils
 {commonActions} = require './utilities'
 sandbox = null
 ROUTER = null
+RENDERED = []
```

Comment:
Yep.  That was left over from when I was trying to figure out how best to unmount components after the spec was done with them.  I see it's also being reset in the beforeEach below, but not used anywhere else.